SADL/AM-CDM-process.sadl

```diff
@@ -1,81 +1,26 @@
-
-	// AM CDM "ProcessSpecification" = IOF/BFO "Plan Specification"
-	described by processSpecification with a single value of type ProcessSpecification.
-
-ProcessSpecification (note "information content entity that has action specifications and objective specifications as parts") is a class, // needed for TIC
-	// AM CDM "ProcessParameters" + "processStandard" + "processProcedure" + "processControlPlan" = IOF/BFO "ActionSpecification" = "information content entity that prescribes what participants shall do in a process"
-	described by processParameters (note "Nominal system settings used when the process is run") with a single value of type ProcessParameters,
+ProcessStepSpecification (note "A prescription of how a planned process step should be performed") is a class, 
 	described by processStandard (note "Standards which process should comply with, ex. AMS2774.") with values of type Document,
 	described by processProcedure (note "Procedure which should be followed when the process is run") with values of type Document,
 	described by processControlPlan (note "Document that describes the methods taken to ensure the quality control of critical inputs to deliver outputs that meet requirements.") with values of type Document,
+	described by processObjective (note "information content entity that prescribes what the outcome of some process should be") with values of type Document,
 
-	// AM CDM added "processObjective" = IOF/BFO "ObjectiveSpecification"
-	described by processObjective (note "information content entity that prescribes what the outcome of some process should be") with values of type Document.
-
-ProcessParameters is a class,
-	described by processParameterID (note "Unique identifier for process parameters.") with a single value of type string.
-	// Subclasses should extend with process-specific process parameters
-
-// Everything that follows should be fleshed out based on domain standards and domain experts. It is provided as an example here as additional use-context
-PreProcess (note "Pre-processing steps.") is a type of ProcessStep.
-
-MaterialPreparation (note "Sifting/blending material preparation.") is a type of PreProcess.
-
-MachinePreparation (note "Operations to prepare an AM machine before a build starts, e.g., cleaning, pre-warming.") is a type of PreProcess.
-
-BuildPlatformPreparation (note "Description of how the build platform was prepared before the AM process (ex. new or recycled).") is a type of PreProcess.
-
-PostProcess (note "Note: The scope of post-processes is broad, which could include assembly as necessary for a part test/certification.") is a type of ProcessStep.
-
-HotIsostaticPressing (note "Hot isostatic processing, which applies heat and pressure simultaneously to objects in an autoclave via an inert gas in such a way as to eliminate internal voids and obtain desired properties.") is a type of PostProcess.
-
-HeatTreatment (note "Heat treatment process.") is a type of PostProcess.
```

Comment:
I don't disagree with these types of ProcessSteps being fleshed out by domain experts, but is having the types in the process SADL file (without domain specific content inside of them) a problem?  I'd kind of rather they were in here and just empty so that people understand what kinds of process steps are out there (non-exhaustive list at this point).

If we do keep these things, I think we should eliminate the concept of pre- and post- (build) processes.  They are all ProcessSteps, the time in which they occur (either before or after something) would be captured in the process history.  e.g. PreProcess should just be ProcessStep and eliminate the concept of PreProcess and PostProcess